venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -165,26 +189,104 @@ function() {
           }
         }
 
+        var publicCommentInvitation = {
```

Comment:
write a new "lightweight" version of this, reserved for the last day of the submission period